shell/roles/hasRole.sh

```
@@ -20,43 +25,30 @@ load_named_args "$@"
 validate_address "$address" "No address specified or it is not an EVM address. Provide the address after the --address flag."
-validate_text "$account" "No account specified. Provide the cast wallet after the --account flag."
+validate_text "$chain" "No chain specified. Provide the chain after the --chain flag."
 
-# Validate environment variables
-echo ""
-echo "Validating environment variables"
-validate_text "$RPC_URL" "No RPC URL specified. Specify the RPC_URL in the $ENV_FILE file."
-validate_text "$CHAIN" "No chain specified. Specify the CHAIN in the $ENV_FILE file."
+# Validate and set forge script flags
+source $SCRIPT_DIR/../lib/forge.sh
+validate_and_set_account "$account" "$ledger"
```

Shellcheck (0.11.0)

[warning] account is referenced but not assigned.

(SC2154)